src/config/config.ts

```diff
@@ -128,11 +126,6 @@ export class ServerConfigHelpers {
         }
     }
 
-    @memoize static parseDisabledTabs(str:string){
```

Comment:
I was having a lot of weird issues having to do w/ circular dependency or some kind of dependency timing issue, having to do with config. So I had to move this function out of config once it referenced the ResultsViewTab enum